lib/std/fs/test.zig

```diff
@@ -49,6 +49,36 @@ fn testReadLink(dir: Dir, target_path: []const u8, symlink_path: []const u8) !vo
     testing.expect(mem.eql(u8, target_path, given));
 }
 
+test "accessAbsolute" {
+    var tmp = tmpDir(.{});
```

Comment:
You need to specifically exclude tests that rely on absolute paths from being tested on WASI target. WASI doesn't have a concept of an absolute path after all.

Reply:
This will also make the tests pass again.